src/components/explore/AlphabetGrouping.tsx

```diff
@@ -1,39 +1,35 @@
-import { getAllCharactersProperty, GroupedData } from "@/utils";
-import { alphabeticalArrangement } from "@/utils/data";
+import { ExploreGroupedData } from "@/utils";
 import Link from "next/link";
 import React from "react";
+import { twMerge } from "tailwind-merge";
 export interface IAlphabetGrouping {
   currentGroup: string;
-  groupedData: GroupedData | never[];
+  groupedData: ExploreGroupedData[];
 }
 const AlphabetGrouping = ({ currentGroup, groupedData }: IAlphabetGrouping) => {
```

Comment:
Same here, `AlphabetNavigation` is fine, same as `MobileAlphabetNavigation`

Reply:
Okay